.gdbinit

```diff
@@ -342,7 +343,11 @@ def fetch_breakpoints(watchpoints=False, pending=False):
                 address = None if is_multiple or is_pending else int(fields[4], 16)
                 is_enabled = fields[3] == 'y'
                 address_info = address, is_enabled
-                parsed_breakpoints[number] = [address_info], is_pending
+                parsed_breakpoints[number] = [address_info], is_pending, ''
+            elif len(fields) >= 5 and fields[1] == 'catchpoint':
+                # only take before comma, but ignore commas in quotes
```

Comment:
Why is the comma thing important here?

Reply:
Because if you want to catch forks, the info break entry looks something like this:
```
1       catchpoint     keep y                      fork, process 25 
```
and only the fork is important, the rest doesn't matter, there can only be one catchpoint for forks and it catches all of them.
but if you wanna catch syscalls for example it looks like this:
```
6       catchpoint     keep y                      syscalls "execve, open"
```
and it matters which syscalls catchpoint got hit since you could have one for open, close etc. and one for another set of syscalls